src/Storage/Services/IApplicationService.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+
+namespace Altinn.Platform.Storage.Services
+{
+    /// <summary>
+    /// This interface describes the required methods and features of a application service implementation.
+    /// </summary>
+    public interface IApplicationService
+    {
+        /// <summary>
+        /// Upload file and save dataElement
```

Comment:
- Wrong summary. 
- Maybe add a description for the return type?

Reply:
Done